include/oneapi/dpl/experimental/kt/esimd_radix_sort_one_wg.h

```diff
@@ -20,12 +20,14 @@
 #include "../../pstl/utils.h"
 
 #include <cstdint>
+#include <cassert>
 
 namespace oneapi::dpl::experimental::kt::esimd::impl
 {
 
-template <typename KeyT, typename InputT, uint32_t RADIX_BITS, uint32_t PROCESS_SIZE, bool IsAscending>
-void one_wg_kernel(sycl::nd_item<1> idx, uint32_t n, uint32_t THREAD_PER_TG, const InputT& input) {
+template <bool _IsAscending, ::std::uint16_t _RadixBits, ::std::uint16_t _DataPerWorkItem,
```

Comment:
May be better to pass KErnelTemplate instance directly into our implementations ?

Reply:
ok